backend/spec/features/admin/products/stock_management_spec.rb

```diff
@@ -42,46 +45,64 @@
     end
 
     it "can create a positive stock adjustment", js: true do
-      adjust_count_on_hand('14')
+      adjust_count_on_hand(variant.id, '14')
       stock_item.reload
       expect(stock_item.count_on_hand).to eq 24
       expect(stock_item.stock_movements.count).to eq 1
       expect(stock_item.stock_movements.first.quantity).to eq 14
     end
 
     it "can create a negative stock adjustment", js: true do
-      adjust_count_on_hand('-4')
+      adjust_count_on_hand(variant.id, '-4')
       stock_item.reload
       expect(stock_item.count_on_hand).to eq 6
       expect(stock_item.stock_movements.count).to eq 1
       expect(stock_item.stock_movements.first.quantity).to eq(-4)
     end
 
     it "can toggle backorderable", js: true do
-      toggle_backorderable(value: false)
+      toggle_backorderable(variant.id, value: false)
 
       click_link "Product Stock"
       within("tr#spree_variant_#{variant.id}") do
         expect(find(:css, "input[type='checkbox']")).not_to be_checked
       end
     end
 
-    def adjust_count_on_hand(count_on_hand)
-      within("tr#spree_variant_#{variant.id}") do
+    def adjust_count_on_hand(variant_id, count_on_hand)
```

Comment:
I ended up not making use of the changes to `adjust_count_on_hand` and `toggle_backorderable` but I think this is a slight improvement anyway so I'd leave it here unless it's considered bad form.